README.md

```diff
@@ -50,16 +50,17 @@ Check out [chris.bolin.co/offline](https://chris.bolin.co/offline) for a simple
 
 `<Online/>`, `<Offline/>`, and `<Detector/>` accept the following props:
 
-| Prop               | Type        | Description                       | Default                        |
-| ------------------ | ----------- | --------------------------------- | ------------------------------ |
-| `polling`          | Obj or Bool | Config for polling fallback [1]   | [see below]                    |
-| `polling.enabled`  | Boolean     | Force polling on or off           | Depends on the browser [1]     |
-| `polling.url`      | String      | URL to pool for connection status | `"https://ipv4.icanhazip.com"` |
-| `polling.interval` | Number      | How often (in ms) to poll         | `5000`                         |
-| `polling.timeout`  | Number      | How long (in ms) before timeout   | `5000`                         |
-| `onChange`         | Function    | Called when connection changes    | none                           |
-| `children` [2]     | Element(s)  | Children **not Detector**         | none                           |
-| `render` [3]       | Func        | Render function **Detector only** | none                           |
+| Prop               | Type        | Description                               | Default                                                                           |
+| ------------------ | ----------- | ---------------------------------         | ------------------------------                                                    |
+| `polling`          | Obj or Bool | Config for polling fallback [1]           | [see below]                                                                       |
+| `polling.enabled`  | Boolean     | Force polling on or off                   | Depends on the browser [1]                                                        |
+| `polling.url`      | String      | URL to pool for connection status         | `"https://ipv4.icanhazip.com"`                                                    |
+| `polling.interval` | Number      | How often (in ms) to poll                 | `5000`                                                                            |
+| `polling.timeout`  | Number      | How long (in ms) before timeout           | `5000`                                                                            |
+| `polling.method`   | String      | method used to poll for connection status | `CONNECT`, `DELETE`, `GET`, `HEAD`, `OPTIONS`, `PATCH`, `POST`, `PUT`, or `TRACE` |
```

Comment:
The last column shows the Default value, so it should just be `HEAD`

And the Description should be `method used to poll for connection status` -> `method to pool for connection status`

Reply:
and why are all the other rows changed? just a editor/prettier auto change?

Reply:
The same reason all the columns were lined up before I made the changes.

Reply:
Yes, the code formatter is called emacs :)

https://www.gnu.org/software/emacs

Reply:
are you going to make the requested changes or just be snarky?